### PR TITLE
Video list

### PR DESCRIPTION
**Pull request by saketkum11**
- videolisting page
- api calls

### CODE REVIEW

**Review by muskaanshah — CHANGES_REQUESTED** [review]
User should be able to view videos of the explore page without being logged in as well.

File: src/Component/Nav/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const Navigation = () => {
+  return (
+    <>
+      <nav class="flex wt-100 bg-blue-5 items-center pd-4 text-color-0 m-y-9">
+        <div class="pd-x-4 cursor">
+          <a href="" class="text-dec text-color-0 text-m">
```
Comment (muskaanshah) [inline src/Component/Nav/Navbar.jsx:6]: suggestion: You can use Link tags here

**Comment by saketkum11** [comment]
> User should be able to view videos of the explore page without being logged in as well.

now I have removed the require auth

**Review by muskaanshah — APPROVED** [review]
Code is modular, easy to understand.
Just one design suggestion - dont keep the title of the video center aligned

File: src/Component/Nav/Navbar.jsx (hunk 1 above)
Comment (saketkum11) [reply src/Component/Nav/Navbar.jsx:6]: okk sure

**Review by sharathnair9999 — APPROVED** [review]
Good use of utility classes. Well written code

**Comment by shubhamkumar648** [comment]
Good use of folder structure and functionality , good to go

**Review by shubhamkumar648 — COMMENTED** [review]
Good use of functionality and folder structure

**Pull request merged**